src/Microsoft.Performance.Toolkit.Engine/EngineCreateInfo.cs

```diff
@@ -38,6 +39,52 @@ public EngineCreateInfo(ReadOnlyDataSourceSet dataSources)
             Guard.NotNull(dataSources, nameof(dataSources));
 
             this.DataSources = dataSources;
+            this.OptionsResolver = new GlobalProcessingOptionsResolver(ProcessorOptions.Default);
+        }
+
+        /// <summary>
+        ///     Specifies an <see cref="IProcessingOptionsResolver"/> to use when processing data sources.
```

Comment:
This method signature (and the one below) does not specify the resolver. The summary for each method and the docstring for the parameter should probably be almost identical. E.g. this should simply state "Specifies the ProcessorOptions to use..."